guides/Implementierungsleitfaden-ISiK-Dokumentenaustausch-401/Einfuehrung/Datenobjekte/Index.page.md

```diff
@@ -9,8 +9,7 @@ Folgende Datenobjekte aus dem Modul [ISiK Basis Stufe 4](https://simplifier.net/
 * [Binary](https://simplifier.net/guide/isik-basis-v4/ImplementationGuide-markdown-Datenobjekte-Datenobjekte_Binary?version=current)
 
 Die Verwendung in diesem Modul bedeutet:
-Wenn dieses Modul auf Datenobjekte aus dem Basismodul referenziert, dann MÜSSEN die referenzierten FHIR-Ressourcen konform zu den Vorgaben aus dem Basismodul sein. 
-Die Referenzierung von im Basismodul spezifizierten Datenobjekten, bedeutet nicht, dass die Interaktionen für die entsprechenden Datenobjekte aus dem Basismodul auch im Modul Dokumentenaustausch bestätigungsrelevant sind.
+Wenn dieses Modul auf Datenobjekte aus dem Basismodul referenziert, dann MÜSSEN die referenzierten FHIR-Ressourcen konform zu den Vorgaben aus dem Basismodul sein.
```

Comment:
```suggestion
Wenn dieses Modul auf Datenobjekte aus dem Basismodul referenziert, dann MÜSSEN die referenzierten FHIR-Ressourcen konform zu Vorgaben an diese Ressourcen aus dem Basismodul sein (Profilkonformität, Interaktionen). Die konkreten Vorgaben zu Interaktionen und Abhängigkeiten zwischen Modulen werden noch präzisiert.
```

Reply:
@alexey-tschudnowsky  und @simoneOnFhir ich habe den ersten Satz erneut geupdated. Da wir hier die "Interaktionen" in der MUSS-Anforderung nicht nennen sollten, solange die Präzisierung in Arbeit ist.

Reply:
problem: DocumentReference sieht auf .actor eine Reference auf Organization vor. Mit der Vorgabe des ersten Satze wird damit allerdings durch die Hintertür eingeführt, dass hier Krankenhäuser/Fachabteilungen/KH-Bereiche eingeführt werden MÜSSEN (da die ISiK Organisation als solche Konzipiert ist). Das kann nicht im Sinne der Spec sein.

Reply:
Du musst den gesamten Abschnitt betrachten:
<img width="1502" alt="image" src="https://github.com/user-attachments/assets/221e1d97-f6b2-457c-9722-cb9cd4bca343">
Hier wird ganz klar NUR auf Patient, Encounter und Binary verwiesen, NICHT auf Organization.
Das Problem ist lediglich, dass der Begriff "referenziert" hier unscharf ist. Wir meinen NICHT alle Ressourcen, die von irgendeiner Resssource dieses Moduls verlinkt werden, sondern wir meinen die drei Ressourcen auf die im Abschnitt darüber Bezug genommen ("referenziert") wird. Vielleicht ist der Begriff "referenziert" hier einfach überladen.

Reply:
du hast recht, @simoneOnFhir , um das klarer zu machen habe ich die Formulierung nochmal angepasst